docs/extending-documentation.md

```diff
@@ -156,18 +206,19 @@ module.exports = {
 };
 ```
 
-You should now see the `LionTabs` instances transformed into your own `LeaTabs`!
+We should now see the `LionButton` instances transformed into our own `ExampleButton`!
+
 
-## Adjust documentation content
+## Remove, add and replace sections
 
-In some cases you don't want to show all examples of how to use a component. Sometimes information is `Lion` specific, or perhaps in your design system you are not allowed to use a certain feature that we documented in `Lion`.
+You might not want to show all examples of how to use a component. Sometimes information is `Lion` specific, or perhaps in your design system people are not allowed to use a certain feature that we documented in `Lion`.
```

Comment:
Minor detail: I think it's a common habit to write Design System in capitals (googling the term confirmed thi suspicion)